src/components/Inputs/input.module.scss

```diff
@@ -252,36 +500,118 @@
             line-height: $text-line-height-3;
             padding: $space-xs $space-l $space-xs $space-s;
 
-            &.with-icon {
+            &.with-icon,
+            &.with-image-icon,
+            &.with-icon-button {
                 padding: $space-xs calc(#{$space-l} + 4px) $space-xs $space-xl;
-            }
 
-            &.with-image-icon {
-                padding: $space-xs calc(#{$space-l} + 4px) $space-xs $space-xl;
-            }
+                &.left-icon {
+                    padding: $space-xs calc(#{$space-l} + 4px) $space-xs
+                        $space-xl;
+                }
 
-            &.with-icon-button {
-                padding: $space-xs calc(#{$space-l} + 4px) $space-xs $space-xl;
+                &.right-icon {
+                    padding: $space-xs $space-xl $space-xs
```

Comment:
guessing this is the large variant? hard to tell with number based class names. large variant seemed like the chevron was too far away from the right edge of the select.

Reply:
will have a look